Sources/Plasma/PubUtilLib/plNetClientComm/plNetClientComm.cpp

```diff
@@ -522,15 +522,15 @@ static void INetCliAuthChangePasswordCallback (
 static void INetCliAuthGetPublicAgeListCallback (
     ENetError                   result,
     void *                      param,
-    const TArray<NetAgeInfo> &  ages
+    const std::vector<NetAgeInfo> & ages
 ) {
     NetCommParam * cp = (NetCommParam *) param;
     
     plNetCommPublicAgeListMsg * msg = new plNetCommPublicAgeListMsg;
     msg->result     = result;
     msg->param      = cp->param;
     msg->ptype      = cp->type;
-    msg->ages.Set(ages.Ptr(), ages.Count());
+    msg->ages       = ages;
```

Comment:
Why not pass by value, then `std::move()` here?

Reply:
`INetCliAuthGetPublicAgeListCallback` is used as a parameter of the function pointer type `FNetCliAuthGetPublicAgeListCallback`, which I didn't feel like committing to removing the reference on...  It's possible(probable) this is the only use though, in which case it would be safe.